pkg/ccl/changefeedccl/schemafeed/schema_feed.go

```diff
@@ -480,36 +480,36 @@ func (tf *schemaFeed) pauseOrResumePolling(ctx context.Context, atOrBefore hlc.T
 	// Always assume we need to resume polling until we've proven otherwise.
 	tf.mu.pollingPaused = false
 
-	if err := tf.targets.EachTableID(func(id descpb.ID) error {
+	canPausePolling, err := tf.targets.EachTableIDWithBool(func(id descpb.ID) (bool, error) {
 		// Check if target table is schema-locked at the current frontier.
 		ld1, err := tf.leaseMgr.Acquire(ctx, frontier, id)
 		if err != nil {
-			return err
+			return false, err
 		}
 		defer ld1.Release(ctx)
 		desc1 := ld1.Underlying().(catalog.TableDescriptor)
 		if !desc1.IsSchemaLocked() {
-			return errors.Newf("desc %d not schema-locked at frontier %s", desc1.GetID(), frontier)
+			return false, nil
 		}
 
 		if atOrBefore.LessEq(frontier) {
-			return nil
+			return true, nil
 		}
 
 		// Check if target table remains at the same version at atOrBefore.
 		ld2, err := tf.leaseMgr.Acquire(ctx, atOrBefore, id)
 		if err != nil {
-			return err
+			return false, err
 		}
 		defer ld2.Release(ctx)
 		desc2 := ld2.Underlying().(catalog.TableDescriptor)
 		if desc1.GetVersion() != desc2.GetVersion() {
-			return errors.Newf("desc %d version changed from version %d to %d between frontier %s and atOrBefore %s",
-				desc1.GetID(), desc1.GetVersion(), desc2.GetVersion(), frontier, atOrBefore)
+			return false, nil
```

Comment:
If we didn't want to lose this log message, I suppose we could put it behind `if log.V(2) { ... }`. I'm not sure what that'll do to your benchmark since we've see `log.V()` show up in profiles before.  I'm not sure howe often we've used this log message for debugging.

Reply:
Good idea, done.